components/ws-manager-bridge/src/cluster-service-server.ts

```diff
@@ -345,7 +341,7 @@ function convertToGRPC(ws: WorkspaceClusterWoTLS): ClusterStatus {
     clusterStatus.setScore(ws.score);
     clusterStatus.setMaxScore(ws.maxScore);
     clusterStatus.setGoverned(ws.govern);
-    clusterStatus.setApplicationCluster(ws.applicationCluster);
+    clusterStatus.setApplicationCluster("");
```

Comment:
For now, we're setting it to the empty value. Once this change lands, we can mark the applicationCluster property as reserved, and remove it.